### ISSUE TITLE
(BUG) SSML cannot be used in Hubitat Rule Manager even though Echo Speaks supports it

### ISSUE THREAD

**Issue by markcanup** [tonesto7/echo-speaks#374]
## Verify the following before opening an trouble issue

**Go over all the following points, and put an `x` in all the boxes that apply.
If you're unsure about any of these, don't hesitate to ask. We're here to help!**

-  [ x] That _OAuth_ is Enabled for the SmartApp under the IDE.
-  [ x] The App/SmartApp and Device Handler/Driver are using the latest code available.
-  [ x] You have reviewed the [Echo Speaks Documentation](https://tonesto7.github.io/echo-speaks-docs) for potential fixes.
-  [ ?] That Both the Apps/SmartApps and Device Handlers/Drivers have been _Published for You_ in the IDE.

---
## About Your Setup
-  Hub Platform (SmartThings or Hubitat): Hubitat
-  How many devices are detected?: 5
-  iOS or Android?: N/A
-  Mobile App Version(Not required):
-  App/SmartApp Version: 3.6.3.1
-  Device Handler/Driver Version: 3.6.3.2
-  Heroku Server Version: 2.6.0

## Expected Behavior
SSML can be used with Echo Speaks devices from actions created within Rule Manager

## Current Behavior
Hubitat Rule Manager strips out markup characters like < and > in input fields, which means SSML cannot be used even though Echo Speaks supports it. 

## Steps to Reproduce (for bugs)
**Provide a link to a live example, or an unambiguous set of steps to reproduce this bug. Include code to reproduce, if relevant**

1. In the Device page for an Echo Speaks device, in the text box below the Speak command, enter the following and click Speak
`<audio src="https://mjcanup01.s3.us-east-2.amazonaws.com/echo-speaks/LylaFood.mp3"/>`
2. This works - the audio file is played on the Echo Speaks device as desired. 

3. In Rule Manager, create an action "Send or Speak a Message"
4. In the Message to send text box, enter the SSML from item 1.
5. The input is wiped, presumably because it contains markup language

6. In Rule Manager, create an action "Run Custom Action"
7. Select a "Speech Device" and then select an Echo Speaks device
8. Select the custom command "Speak" and then parameter type "String"
9. in the "String to speak" box, enter the SSML from item 1.
10. The input is wiped, presumably because it contains markup language

11. Repeat steps 6-8 above
12. In the "String to speak" box, enter 
`&lt;audio src="https://mjcanup01.s3.us-east-2.amazonaws.com/echo-speaks/LylaFood.mp3"/&gt;`
13. This time the input is transformed into the SSML in item 1. and accepted by Rule Manager
14. When running the action, the Echo Speaks device uses TTS to say "audio SRC equals HTTPS ..." and so forth, and does not play the audio file as desired.

## Context
**How has this issue affected you? What are you trying to accomplish?
**Providing context helps us come up with a solution that is most useful in the real world**
I use Echo Speaks extensively to speak various messages to my family about doors being open, washing machine finished, reminders to children to get ready for bed, etc. I am trying to replace some of the TTS with using audio recordings of my own voice. I realize this issue is really with Rule Manager input boxes, and not with Echo Speaks, but since the Echo Speaks documentation says that SSML is supported, I am posting here. If there is some kind of workaround to be able to play personally created MP3 files on my Echo Speaks devices a different way, I would be happy to use it.

Thanks!

---
Please include a copy of any relevant log output to assist in tracking down the bug

**Comment by markcanup** [comment]
I realized that the reason Echo Speaks was reading the text of the code in step 14 above was that the &lt; and &gt; were actually in the parameter; it was displaying as I expected in the UI but under the covers it was passing &lt; and &gt; to the Alexa routines and it wasn't being interpreted as SSML.

I figured out a workaround. I set 
`<audio src='https://mjcanup01.s3.us-east-2.amazonaws.com/echo-speaks/LylaFood.mp3'/`
as a global variable called TEST_AudioFile. Not sure if I had to do this, but I created a connector for the variable to set it using a dashboard since the string checking is different there than in the RuleManager UI. Then I called the speak command from RuleManager with 
`%TEST_AudioFile%>`
Since I put the opening < in the variable and the closing > in the parameter call, the string checking didn't detect that I had embedded SSML. Since Echo Speaks supports SSML, then once I finally got the SSML to be able to be passed to it from Rule Manager, it worked as intended.

It's unfortunate people have to go to such lengths to use SSML in Hubitat. 

Thanks...
